### DISCUSSION TITLE
Custom lastmod and date attribute names not working

### DISCUSSION THREAD

**Discussion by jmatthewpryor** [estruyf/vscode-front-matter#278] [General]
Hi

I've followed the documentation to try & get custom publish and last mod dates working - so far without success

My workspace editor settings appear correct

![Screen Recording 2022-03-05 at 03 53 11 pm](https://user-images.githubusercontent.com/850570/156868524-d0476d38-7913-4509-9351-6c9f30dad969.gif)

And I have edits frontmatter.json in my project also

```
"frontMatter.taxonomy.dateField": "publishedAt",
"frontMatter.taxonomy.modifiedField": "lastMod",
```

However, the FrontMatter editor fails to pick up those field value and sets a field "date" when I use the Dashboard to update

I am running on the Beta

Any tips appreciated

**Comment by estruyf** [comment]
Thanks @jmatthewpryor this should be optimized, as there are indeed some issues. These two settings came from the time when Front Matter didn't support content types yet. 

Since FM uses content types, this can create some conflicts. To explain:

- `frontMatter.taxonomy.dateField`: This setting is still in use for rendering the date field on your dashboard.
- `frontMatter.taxonomy.modifiedField`: This setting is still used for automatically updating the modified date on save (when the auto-update modified date setting is set)

I've been thinking about it, and it might be best to deprecate these settings and introduce two new properties for the content type fields:

- `isPublishDate`: defines the publish date
- `isModifiedDate`: define the modified date

The advantage of these new properties is that the content dashboard can use multiple field types.

**Reply by estruyf** [reply]
You can follow the status in issue #281

**Reply by jmatthewpryor** [reply]
sorted by #281 - thanks
